eth-rpc-adapter/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:14-alpine
+
+ENV NODE_ENV=production
+
```

Comment:
is it required field or just get as it was in example code which shared by Bryan ?

Reply:
we want to have this for prod image